snippets/PowerShell.json

```diff
@@ -770,7 +770,16 @@
             "\t$0",
             "}"
         ],
-        "description": "function definition snippet"
+        "description": "function definition snippet that contains a param block"
+    },
+    "function-NoParam": {
+        "prefix": "function-noparam",
+        "body": [
+            "function ${1:FunctionName} (${2:OptionalParameters}) {",
+            "\t$0",
+            "}"
+        ],
+        "description": "function definition snippet that does not contain a param block and is more typically seen in C#"
```

Comment:
How about we describe this as just `function definition snippet with inline parameters`?

Reply:
Could do but I think this is more informative, and explains in not too much info where a user may see this syntax elsewhere

Reply:
Well, my main objection is that the last bit about "and is more typically seen in C#" reads a bit too "opinionated" IMO.  :-)  Maybe just remove that bit?